commit 7c697a5d187b6f8ed0c0eb44d7ae3cbd2074e165
Author: Xu Fasheng <fasheng.xu@gmail.com>
Date:   2015-09-09 07:28:00 +0800

    loader: fix block issue under go1.5
    
    Run modules as the package importing order to ensure
    keybinding module runs after inputdevice.
    
    Change-Id: Ie2a92b31346a99435414bd70139e78bb3b55735f

diff --git a/loader.go b/loader.go
index 15da0e3..b5c9e58 100644
--- a/loader.go
+++ b/loader.go
@@ -51,7 +51,7 @@ func Register(newModule *Module) {
 		logger.Error("can't register an incomplete module " + newModule.Name)
 		return
 	}
-	modules = append([]*Module{newModule}, modules...)
+	modules = append(modules, newModule)
 }
 
 func Start() {
